src/esbmc/esbmc_parseoptions.cpp

```diff
@@ -345,7 +345,9 @@ void esbmc_parseoptionst::get_command_line_options(optionst &options)
     options.set_option("no-slice", true);
   }
 
-  if (cmdline.isset("smt-thread-guard") || cmdline.isset("smt-symex-guard"))
+  if (
+    cmdline.isset("smt-thread-guard") || cmdline.isset("smt-symex-guard") ||
+    cmdline.isset("smt-symex-assert"))
```

Comment:
Can we enable `smt-during-symex` by default when we detect the use of these flags?

Reply:
Yes, this might be a better solution!

Reply:
Implemented as suggested.